02_configure_host.sh

```diff
@@ -198,3 +198,15 @@ sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name vbmc ${POD_NAM
 sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name sushy-tools ${POD_NAME_INFRA} \
      -v "$WORKING_DIR/virtualbmc/sushy-tools":/root/sushy -v "/root/.ssh":/root/ssh \
      "${SUSHY_TOOLS_IMAGE}"
+
+# Installing the openstack/ironic clients on the host is optional
+# if not installed, we use an alias to openstackclient.sh which
+# runs the clients in a container (metal3-io/ironic-client)
+if ! command -v openstack; then
+  if ! grep -q "^alias openstack" ~/.bashrc; then
+    echo "alias openstack=\"${SCRIPTDIR}/openstackclient.sh\"" >> ~/.bashrc 
```

Comment:
Not everyone uses bash though. Also, if $SCRIPTDIR changes, their alias won't get updated.

I think just printing some message with the path to openstackclient.sh at the very end of metal3-dev-env run would be better.

Reply:
Bash is the default shell on many/most platforms these days so I was trying to reduce friction in the common case - I can grep for the entire string though to catch someone changing the $SCRIPTDIR (this seems unlikely but it's a possible corner-case for sure).

We can definitely also print some message and/or document the running the script directly - this was really just a shim to ensure backwards compatibility and avoid breaking existing users who are used to typing `openstack foo`